### PR TITLE
ansible tests: Add test for the neighbour MAC change update

### PR DESCRIPTION
**Pull request by andriymoroz-mlnx**
This is the test for the issue https://github.com/Azure/SONiC/issues/19

### CODE REVIEW

File: ansible/roles/test/tasks/sonic.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+- name: Neighbour tests
```
Comment (lguohan) [inline ansible/roles/test/tasks/sonic.yml]: rename to "Neighbor mac change test"

File: ansible/roles/test/tasks/sonic.yml [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+  include: neighbour.yml
```
Comment (lguohan) [inline ansible/roles/test/tasks/sonic.yml]: rename to "neighbor-mac.yml"

File: ansible/roles/test/files/neighbour/neigh_test.sh [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+# get neighbour mac from SAI DB for given ip
+# $1 - ip of the neighbour
+# ret: 1 - not found
+#      0 - success
+function get_neighbour_mac()
+{
+    # convert ip to hex format (20.0.0.2 -> 14000002)
+    iface_ip_hex=`printf '%02X' ${iface_ip//./ }`
+    # grep redis DB to find a key holding data for the neighbour with this IP
+    key=`ssh ${dut_user}@${dut_host} "bash -c 'docker exec database redis-cli -n 1 KEYS ASIC* ' | grep -i ${iface_ip_hex}"`
```
Comment (andriymoroz-mlnx) [inline ansible/roles/test/files/neighbour/neigh_test.sh]: agree about ssh from host to DUT - it relies on keys which needs some pre-setup.
and making redis-cli listen to management port will also require some configuration
what about refreshing the known_hosts on DUT before the test?
Or do all ssh work from the ansible_host - it has login/password for both host and DUT and will handle any issues?

Comment (andriymoroz-mlnx) [inline ansible/roles/test/files/neighbour/neigh_test.sh]: Rewrote test to use ansible scripts only
Changes for the new DB format are in progress...

File: ansible/roles/test/files/neighbour/neigh_test.sh [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    #echo $key; echo
+    # NEIGHBOUR_DST_MAC_ADDRESS attribute code is 0, so using key 00000000
+    mac=`ssh ${dut_user}@${dut_host} "bash -c 'docker exec database redis-cli -n 1 HGET ${key} 00000000'"`
```
Comment (andriymoroz-mlnx) [inline ansible/roles/test/files/neighbour/neigh_test.sh]: Updated the test. Tested on the recent sources

**Pull request merged**